src/AppBundle/Helper/AutomatedEditsHelper.php

```diff
@@ -24,13 +27,23 @@ class AutomatedEditsHelper
     /** @var ContainerInterface The service container. */
     private $container;
 
+    /** @var Project $metaProject */
+    private $metaProject;
+
+    /** @var CacheItemPoolInterface The cache. */
```

Comment:
"The cache." doesn't really help document anything since the variable is named $cache and the type is clearly a cache

Reply:
Hehe I agree. This is actually just to get around PHPCS from yelling at us. The sniffing rules are probably a little too strict. Anyway I expanded it to "The caching interface" :)